lib/node_modules/@stdlib/blas/base/isamax/src/isamax_f.c

```diff
@@ -18,17 +18,66 @@
 
 #include "stdlib/blas/base/isamax.h"
 #include "stdlib/blas/base/isamax_fortran.h"
+#include "stdlib/blas/base/shared.h"
+#include "stdlib/blas/base/xerbla.h"
+#include "stdlib/blas/base/scopy.h"
+#include "stdlib/strided/base/min_view_buffer_index.h"
+#include <stdlib.h>
 
 /**
 * Finds the index of the first element having the maximum absolute value.
 *
 * @param N        number of indexed elements
 * @param X        input array
-* @param strideX  stride length
+* @param strideX  X stride length
 * @return         index value
 */
-int c_isamax( const int N, const float *X, const int strideX ) {
-	int idx;
+CBLAS_INT API_SUFFIX(c_isamax)( const CBLAS_INT N, const float *X, const CBLAS_INT strideX ) {
+	CBLAS_INT stride;
+	CBLAS_INT idx;
+	float *copyX;
+
+	if( strideX < 0 ) {
```

Comment:
@aman-095 Would you mind copying over the changes from `isamax_cblas.c` to this file? Thanks!